pkg/distro/defs/fedora/distro.yaml

```diff
@@ -167,15 +167,18 @@
     additional_dracut_modules:
       - "net-lib"
     squashfs_rootfs: true
-    condition:
-      version_less_than:
-        "42":
-          # config is fully replaced
+    conditions:
+      "f41 uses ifcfg in dract but already a squashfs rootfs":
+        when:
+          version_equal: "41"
+        override:
           additional_dracut_modules:
             - "ifcfg"
           squashfs_rootfs: true
-        "41":
-          # config is fully replaced
+      "f40 and lower uses ifcfg in dracut and no squashfs rootfs":
```

Comment:
'no squashfs rootfs' is misleading. 'squashfs+ext4 rootfs' is really what is created.